module/entities/helper.js

```diff
@@ -0,0 +1,208 @@
+                    }
+                }
+
+                if ( locationDeatails.codes.length > 0 ) {
+                    let bodyData = {
+                        "code" : locationDeatails.codes
+                    } 
+                    let entityData = await userService.locationSearch( bodyData, formatResult );
+                    if ( entityData.success ) {
+                        entityInformation =  entityInformation.concat(entityData.data);
+                    }
+                }
+
+                if ( !entityInformation.length > 0 ) {
+                    throw {
+                        message : CONSTANTS.apiResponses.ENTITY_NOT_FOUND
+                    } 
+                }
+
+                return resolve({
+                    success : true,
+                    message : CONSTANTS.apiResponses.ENTITY_FETCHED,
+                    data : entityInformation
+                });
+
+            } catch(error) {
+                return resolve({
+                    success : false,
+                    message : error.message
+                });
+            }
+        })
+    }
+
+
+      /**
+     * Get Observation document based on filtered data provided.
+     * @method
+     * @name observationDocuments
+     * @param {Object} [findQuery = "all"] -filter data.
+     * @param {Array} [fields = "all"] - Projected fields.
+     * @returns {Array} - List of observations.
+     */
+
+      static observationDocuments(findQuery = "all", fields = "all") {
+        return new Promise(async (resolve, reject) => {
+            try {
+                let queryObject = {};
+
+                if (findQuery != "all") {
+                    queryObject = _.merge(queryObject, findQuery)
+                }
+
+                let projectionObject = {};
+
+                if (fields != "all") {
+                    fields.forEach(element => {
+                        projectionObject[element] = 1;
+                    });
+                }
+
+                let observationDocuments = await database.models.observations
+                    .find(queryObject, projectionObject)
+                    .lean();
+
+                return resolve(observationDocuments);
+            } catch (error) {
+                return reject(error);
+            }
+        });
+    }
+
+
+
+     /**
+   * Implement find query for entity
+   * @method
+   * @name entityDocuments
+   * @param {Object} [findQuery = "all"] - filter query object if not provide 
+   * it will load all the document.
+   * @param {Array} [fields = "all"] - All the projected field. If not provided
+   * returns all the field
+   * @param {Number} [limitingValue = ""] - total data to limit.
+   * @param {Number} [skippingValue = ""] - total data to skip.
+   * @returns {Array} - returns an array of entities data.
+   */
+
+     static entityDocuments(findQuery = "all", fields = "all", limitingValue = "", skippingValue = "",sortedData = "") {
```

Comment:
@Prajwal17Tunerlabs move entityDocuments to databaseQueries folder

Reply:
have moved above code to databaseQueries